### DISCUSSION TITLE
Fehler in Hover Texten

### DISCUSSION THREAD

**Discussion by Shark-Fresh** [rjcncpt/StarCitizen-Deutsch-INI#370] [Übersetzungsvorschläge]
Texte, die bereits schon gut übersetzt wurden, werden dann wieder falsch oder gar nicht übersetzt?!?!
Beispiel Fahrstuhl Reclaimer...  War einige Tage richtig und gut, nun wieder teils englisch und teils deutsch, warum?
Die Anzeige "Schiff" wird nun auch bei anderen Raumschiffen angezeigt statt wie bisher "Pilotensitz betreten" o.ä.
Bei manchen Schiffen zeigt es nun auch zweimal "Co-Pilot betreten" an o.ä. ?!
Ich hoffe man geht bei den Übersetzungen schon einigermaßen systematisch vor und lässt nicht immer irgendwelche KIs drüber jagen?!
Bin zwar dankbar dass dies überhaupt gemacht wird, aber bitte richtig und nicht dass man gutes wieder schlechter macht, danke.

**Comment by rjcncpt** [comment]
Hey. Kannst du Screenshots beisteuern? KI lassen wir nur noch bei Beschreibungen durchlaufen. UI oder ähnliches nicht mehr. Also wenn du uns visuelle Beispiele zeigen kannst, wäre das perfekt.

**Comment by rjcncpt** [comment]
Was das Thema "Schiff" betrifft, dass sollte vorhin korrigiert worden sein. Das wurde im Knebel Discord als fehler gemeldet. Hier sollte wieder Betreten stehen. Was den Fahrstuhl betrifft, da brauchen wir mal einen Screenshot der den Fehler zeigt

**Comment by rjcncpt** [comment]
Ist behoben. Zur Erklärung: das war tatsächlich Absicht da wir erst einmal eine vernünftige Übersetzung für „Salvage Hold“ finden wollten.
